source/fundamentals/typescript.txt

```diff
@@ -145,4 +145,4 @@ section.
 Limitations of Driver Version {+version+}
 ---------------------------------
 
-.. include:: includes/limitations/{+version+}.rst
+.. include:: includes/limitations/limits.rst
```

Comment:
Issue: Rather than renaming the file to `limits`, could the `4.3.rst` file be copied to a `4.4.rst` file? I believe this helps docs maintainers understand how content applies to different versions with less research. [Here is the initial comment that requested that this structure be used for limitations files](https://github.com/mongodb/docs-node/pull/275#discussion_r788054575).